agent/.gitignore

```diff
@@ -8,3 +8,4 @@ log/*.log
 /build/
 /release/
 /vendor/
+.rspec-local
```

Comment:
Doesn't git pick it up from `../.gitignore`? I don't think this is needed here or in the cli/server.